As we don't have SVG support we disable the search for the converter.

Author: Alexander Grund (TU Dresden)

diff -aur a/configure.ac b/configure.ac
--- a/configure.ac	2021-07-16 15:16:15.774630738 +0200
+++ b/configure.ac	2021-07-16 15:15:10.138814059 +0200
@@ -49,7 +49,7 @@
 AM_CONDITIONAL(ALLOW_RENDERING, test "x$allow_rendering" = "xyes")
 
 symbolic_encode_sizes="16x16 24x24 32x32 48x48 64x64 96x96"
-AC_PATH_PROG([GTK_ENCODE_SYMBOLIC_SVG], [gtk-encode-symbolic-svg], [false])
+GTK_ENCODE_SYMBOLIC_SVG="false"
 if test "x$GTK_ENCODE_SYMBOLIC_SVG" = "xfalse"; then
    symbolic_encode_sizes=""
 fi
